_posts/2021-02-25-migrating-existing-apps-to-quarkus-with-mta..adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Migrating existing applications to Quarkus with Migration Toolkit for Applications
+date: 2021-02-25
+tags: mta
+synopsis: Migrating existing Java applications, like Spring Boot ones, to Quarkus by using the Migration Toolkit for Applications
+author: mperezco
```

Comment:
The author image is missing: https://quarkusio-quarkusio-github-io-preview-pr-907.surge.sh/blog/migrating-existing-apps-to-quarkus-with-mta/

You need to add yourself to the `_data/authors.yaml` file

Reply:
Ah I see you created https://github.com/quarkusio/quarkusio.github.io/pull/906 for this, but unfortunately it needs a rebase. Can you close that PR and bring the commit over to this PR?